eng/test-configuration.json

```diff
@@ -5,6 +5,8 @@
   "retryOnRules": [
     { "testAssembly": { "regex": "Aspire.EndToEnd.*" }, "failureMessage": { "regex": "App run failed" } },
     { "testAssembly": { "regex": "Aspire.Npgsql.EntityFrameworkCore.PostgreSQL.*" } },
-    { "testAssembly": { "regex": "Aspire.Microsoft.EntityFrameworkCore.SqlServer.*" } }
+    { "testAssembly": { "regex": "Aspire.Microsoft.EntityFrameworkCore.SqlServer.*" } },
```

Comment:
Are we sure this assembly needs retry? Isn't it unit tests only?

Reply:
These two can be dropped. We had added retries for old issues which have since been fixed.
https://github.com/dotnet/aspire/commit/4976ee1366bb8119703be6d0a15f997975ea732b

Reply:
My point is more about:
- When to use `testAssembly` vs `testName`
- That `"Aspire.Npgsql.EntityFrameworkCore.PostgreSQL.*"` is not a regex, or at least it's doing what we assume it tries to do by chance (the dot can be anything and the `*` appears right after a dot, double points for luck here).